app/views/pages/information.html.erb

```diff
@@ -61,10 +55,6 @@
     </ul>
   </div>
   <div class="social-links">
-    <a href="https://twitter.com/thepracticaldev" target="_blank" rel="noopener"><img src="<%= asset_path("twitter-logo.svg") %>" alt="DEV Twitter profile"></a>
-    <a href="https://github.com/thepracticaldev" target="_blank" rel="noopener"><img src="<%= asset_path("github-logo.svg") %>" alt="DEV GitHub profile"></a>
-    <a href="https://instagram.com/thepracticaldev" target="_blank" rel="noopener"><img src="<%= asset_path("instagram-logo.svg") %>" alt="DEV Instagram profile"></a>
-    <a href="https://facebook.com/thepracticaldev" target="_blank" rel="noopener"><img src="<%= asset_path("facebook-logo.svg") %>" alt="DEV Facebook profile"></a>
-    <a href="https://twitch.tv/thepracticaldev" target="_blank" rel="noopener"><img src="<%= asset_path("twitch-logo.svg") %>" alt="DEV Twitch profile"></a>
+    <%= render partial: "layouts/social_media" %>
```

Comment:
Note here: This was 35px initially, as soon as I moved to using the partial and inline_svg it made 24px, I think it looks okay. but maybe @nickytonline can help me to figure out how to override it to 35px on the information page. I tried to do it with css by setting a new height and width on `.social-links .social-media-link`. I also tried setting a. width and height on inline_svg

Reply:
I also tried this as per the [inline_svg documentation](https://github.com/jamesmartin/inline_svg) :

```
<%= inline_svg_tag("#{social_media_type}.svg", aria: true, title: social_media_type.capitalize, class: "crayons-icon social-media-svg") %>
```
```
.social-links .social-media-svg {
  width: 35px;
  height: 35px;
}
```

Reply:
@Ridhwana, I checked out your PR locally, searched the project for `.social-media-svg`, and it appears nowhere.

As well, on your branch

```
<%= inline_svg_tag("#{social_media_type}.svg", aria: true, title: social_media_type.capitalize, class: "crayons-icon") %>
```

is missing the `social-media-svg class`.

Reply:
@nickytonline it may have not been clear from my above message, but I popped the code here in the comments  because I didn't commit it as it didn't work. If you'd prefer, I can quickly add it to the branch before I go to bed if you think this is the correct route and you'd like to test it out

Reply:
@nickytonline so I just converted the comment to code, so you should find the class and css there now, apologies for the misunderstanding.